docs/docs-beta/docs/dagster-plus/access/authentication/azure-ad-sso.md

```diff
@@ -75,7 +75,7 @@ dagster-cloud organization settings saml upload-identity-provider-metadata <path
    --url https://<organization_name>.dagster.cloud
 ```
 
-## Step 4: Create a test user \{#test-user}
+## Step 4: create a test user \{#test-user}
```

Comment:
@erinkcochran87 I found we can support uppercase following `:` with the `indicators` parameter in our style -- I made these changes before updating the rule. What is our stance on this?